js/ui/tag_box.js

```diff
@@ -864,24 +865,66 @@ const TagBox = SelectBox.inherit({
 
     _renderTags: function() {
         const d = new Deferred();
+        let plainDataUsed = false;
```

Comment:
Don't u think, it's good idea to use `is` prefix for boolean vars? like `isPlainDataUsed`